src/main/java/com/github/ogomezso/kafka/connect/soap/source/SoapSourceTaskConfig.java

```diff
@@ -0,0 +1,88 @@
+
+public class SoapSourceTaskConfig extends AbstractConfig {
+
+  public static final String CONNECTION_TIMEOUT = "connectionTimeOut";
+  public static final String ENDPOINT_URL = "endpointUrl";
+  public static final String POLL_INTERVAL = "pollInterval";
+  public static final String PORT_NAME = "portName";
+  public static final String REQUEST_MSG_FILE = "requestMessageFile";
+  public static final String REQUEST_TIMEOUT = "resquestTimeOut";
+  public static final String SERVICE_NAME = "serviceName";
+  public static final String SOAP_ACTION = "SOAPAction";
+  public static final String TARGET_NAMESPACE = "targetNameSpace";
+  public static final String TOPIC = "topic";
+  private static final String CONNECTION_TIMEOUT_DOC = " SOAP Service Connection timeout in milliseconds";
+  private static final String ENDPOINT_URL_DOC = "Endpoint url for a service";
+  private static final String POLL_INTERVAL_DOC = "Time between service calls in milliseconds";
+  private static final String PORT_NAME_DOC = "Port Name for a service";
+  private static final String REQUEST_MSG_FILE_DOC = "Absolute path to xml file containing the service message";
+  private static final String REQUEST_TIMEOUT_DOC = "SOAP Request Tieout in Milliseconds";
+  private static final String SERVICE_NAME_DOC = "Service Name for SOAP will be invoked";
+  private static final String SOAP_ACTION_DOC = "SOAP Action for a message";
+  private static final String TARGET_NAMESPACE_DOC = "Target Namespace for the SOAP Client";
+  private static final String TOPIC_DOC = "Topic to send events to";
+
+  private final Long connectionTimeOut;
+  private final String endpointUrl;
+  private final String portName;
+  private final Long pollIntervalSeconds;
+  private final File requestMsgFile;
+  private final Long requestTimeout;
+  private final String serviceName;
+  private final String soapAction;
+  private final String targetNameSpace;
+  private final String topic;
+
+  public SoapSourceTaskConfig(Map<?, ?> originals) {
+    super(config(), originals);
+    connectionTimeOut = this.getLong(CONNECTION_TIMEOUT);
+    endpointUrl = this.getString(ENDPOINT_URL);
+    portName = this.getString(PORT_NAME);
+    pollIntervalSeconds = this.getLong(POLL_INTERVAL);
+    requestMsgFile = ConfigUtils.getAbsoluteFile(this, REQUEST_MSG_FILE);
+    requestTimeout = this.getLong(REQUEST_TIMEOUT);
+    serviceName = this.getString(SERVICE_NAME);
+    soapAction = this.getString(SOAP_ACTION);
+    targetNameSpace = this.getString(TARGET_NAMESPACE);
+    topic = this.getString(TOPIC);
+  }
+
+  public static ConfigDef config() {
+
+    return new ConfigDef()
+        .define(CONNECTION_TIMEOUT, Type.LONG, 30000, Importance.MEDIUM, CONNECTION_TIMEOUT_DOC)
+        .define(ENDPOINT_URL, Type.STRING, Importance.HIGH, ENDPOINT_URL_DOC)
+        .define(PORT_NAME, Type.STRING, Importance.HIGH, PORT_NAME_DOC)
+        .define(POLL_INTERVAL, Type.LONG, 60, Importance.HIGH, POLL_INTERVAL_DOC)
```

Comment:
perhaps we should increase the poll interval to 60000. I missed it on previous PR (changed from seconds to milliseconds)

Reply:
Yes, but then also reduce request timeout? Would be great to get rid of the error that poll interval has to be bigger than request timeout.

Reply:
I think that putting to 60k we are maintaining the same ration beetween request timeout and poll interval is just a matter os measure (seconds vs milliseconds). Connection timeout is set to 30sg (30k ms) and with the change we will set the poll interval to 60s (60ms).

Maybe you are thinking on a better ratio, feel free to set the values as you think correct :)